workspace/apps/pidp/src/scss/material/components/_mat-button.scss

```diff
@@ -7,7 +7,7 @@
 .mat-stroked-button {
   &.mat-button-base.mat-primary {
     // Stroked button default has a grey border
-    border-color: theme-palette(blue);
+    border-color: theme-palette(purple);
```

Comment:
We're only doing the primary colour the secondary colour has not been officially chosen

Reply:
reverted secondary colour